src/containers/Protocol.js

```diff
@@ -12,22 +12,23 @@ class Protocol extends Component {
 
   componentWillMount() {
     if (!this.props.protocol.protocolLoaded) {
-      this.props.loadProtocol()
+      this.props.loadProtocol(this.props.stageId);
```

Comment:
This is how you get the info from the router initially. But if we move away from using the router this is not necessary either.